src/Service/AbstractService.php

```diff
@@ -40,11 +41,17 @@ public function __construct(SoapClient $soapClient, RequestGeneratorInterface $r
      * @param array $message the message to be passed to the operation.
      *
      * @return object the raw response.
+     *
+     * @throws ServiceException
```

Comment:
Maybe an description say when this is throw: On soap exceptions.

Reply:
Fixed on bdf4adb